interface/patient_file/birthday_alert/birthday.inc.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
For the picture above, place it in public/images
And use the following variables for path when using it in code: https://github.com/openemr/openemr/blob/master/interface/globals.php#L172-L176
(also make sure it is not copyrighted or anything like that)